doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
 * ITU-T Rec. G.8264 that describes the specification of Ethernet Synchronization Messaging Channel (ESMC)
 SyncE architecture minimally requires replacement of the internal clock of the Ethernet card by a phase locked loop in order to feed the Ethernet PHY. 
 
+
+<a name="3.5.7"></a>
+ ### 3.5.7 Kubernetes Networking Semantics
+The support for traditional network orchestration is more or less non existing in Kubernetes proper. Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastrutcure as a Service (Iaas) infrastucture component. There is no orchetration API like Neutron and no way to instantiate network services such as L3aaS and LBaaS like you can do in Openstack.
+
+Kubernetes networking can be divided into two parts, built in network functinality available through the pod's mandatory primarty interface and network functionality avaiable through the pod's optional secondary inerfaces.
+
+<a name="3.5.7.1"></a>
+#### 3.5.7.1 Built in Kubernetes network functionality 
+Kubernetes currently only allows for one network, the *cluster* network and one network attachment for each pod. All pods and containers have an *eth0* interface, this interface is created by kubernetes at pod cration and attached to the *cluster* network. All communication to and from the pod is done using this interface. To only allow for one interface in a pod removes the need for traditional networking tools such as *VRFs* and complicated routing tables inside the pod network namespace. 
```

Comment:
convert the "," after "All pods and containers have an *eth0* interface" either to a period (".") or a semicolon (";").

Reply:
nit: cration -> creation

Reply:
done